app/src/ui/app.tsx

```diff
@@ -612,6 +614,29 @@ export class App extends React.Component<IAppProps, IAppState> {
     this.props.dispatcher.openInBrowser(compareURL)
   }
 
+  private showBranchOnGitHub() {
+    const htmlURL = this.getCurrentRepositoryGitHubURL()
```

Comment:
Most of the logic in here could be shared with `compareBranchOnDotcom()`

Reply:
@ericboucher great catch 🔝 @tidy-dev 👍